Gulpfile.js

```diff
@@ -4,29 +4,40 @@ const gulp = require('gulp');
 const sass = require('gulp-sass');
 
 const paths = {
-  scss: [
-    './core/**/*.scss',
-    './contrib/*.scss',
-  ]
+  markup: {
+    src: './contrib/**/*.html',
+  },
+  styles: {
+    src: [
+      './contrib/**/*.scss',
+      './core/**/*.scss',
+    ],
+    dest: './contrib/',
+  }
 };
 
-gulp.task('serve', ['sass'], function() {
+function styles() {
+  return gulp.src(paths.styles.src)
+    .pipe(sass({includePaths: bourbon}))
+    .pipe(gulp.dest(paths.styles.dest));
+};
+
+function serve(done) {
```

Comment:
What do you use this callback for?

Reply:
I have no idea! I was banging my head because it wasn’t working initially, but once I added this, it worked. I _think_ it is because this is a synchronous task?